### ISSUE TITLE
Add absolute path to Spiral/Database/Injection/Fragment in alterColumn function

### ISSUE THREAD

**Issue by mrakolice** [spiral/migrations#8]
![image](https://user-images.githubusercontent.com/4227668/72306450-d26aee80-36c2-11ea-8c8b-7389de2153cc.png)

Without leading slash composer try to find this class in Migration namespace instead of global.

**Comment by mrakolice** [comment]
I found monkey patching solution for this particular usecase, spiral/reactor/src/Serializer#117-119 need to be rewrite as

```
 if (is_object($value)) {
            return '\\' . var_export($value, true);
}

```

**Comment by mrakolice** [comment]
Or  just add __set_state function to Spiral/Database/Injection/Fragment class

**Comment by wolfy-j** [comment]
Thank you for the report. Will be fixed today.

**Comment by wolfy-j** [comment]
Please check the v2.7.1 version of `spiral/database`. Let me know if you still need the Reactor change.

**Comment by mrakolice** [comment]
This problem is still in project. So I think, that Reactor need to change a little bit.

**Comment by wolfy-j** [comment]
I would greatly appreciate your PR to Reactor. If not I’ll check next week.

**Comment by vvval** [comment]
At least it would be great to have a failing test

**Comment by mrakolice** [comment]
Here example of failing entities.

https://github.com/mrakolice/migrations/tree/error-with-spiral-reactor

my composer.lock:

cycle/orm v1.1.18
spiral/database v2.6.10
spiral/reactor v2.2.2

**Comment by mrakolice** [comment]
If I  try to do

```
if (is_object($value) && method_exists($value, '__set_state')) {
            return '\\' . var_export($value, true);
}
```
in spiral/reactor/Serializer.php, then function BaseTest->assertSameAsInDb start  working unexpectedly

![image](https://user-images.githubusercontent.com/4227668/72694964-a0ee9900-3b82-11ea-90f9-2cf72261fee4.png)

But in  database I can see changes (column type change from `integer`  to `bigint`)

**Comment by vvval** [comment]
@wolfy-j I assume that the second error in tests is related directly to the migrations package

**Comment by vvval** [comment]
The reactor has been fixed in the [v2.2.3](https://github.com/spiral/reactor/releases/tag/v2.2.3) release

**Comment by mrakolice** [comment]
> The reactor has been fixed in the [v2.2.3](https://github.com/spiral/reactor/releases/tag/v2.2.3) release

Thanks! I am waiting for migrations package fix.

**Comment by vvval** [comment]
@mrakolice I've added your breaking input as a PR to the [`cycle/migrations`](https://github.com/cycle/migrations/pull/7) itself

**Comment by vvval** [comment]
By the way, what is the purpose of changing PK column type? Also, I assume that the correct alter change should contain also `primary=true` annotation attribute

**Comment by mrakolice** [comment]
@vvval yeah, you absolutely right about ``primary=true``, but this solution works only with fresh database, created with CodeFirst. This problem (with sequences) starts when you try to migrate from existing schema, in my case created by another orm(eloquent).

I think, this problem will be exist when altering fields with other sequences, rather than PK, but I don't testing it.

**Comment by wolfy-j** [comment]
Val, can you check the default value of that column and see what needs to be written into Column in order to keep the type persistent.

**Comment by wolfy-j** [comment]
Would readonly schema help in this case?

**Comment by vvval** [comment]
@mrakolice see above please

**Comment by mrakolice** [comment]
What should I do? Make database schema (in rdbms) readonly or what?

**Comment by wolfy-j** [comment]
Check readonlySchema annotation, it won’t generate migration for your table: https://cycle-orm.dev/docs/annotated-entity

**Comment by mrakolice** [comment]
This is not my solution, unfortunately.
